rainbow-cake-core/src/main/java/co/zsmb/rainbowcake/base/RainbowCakeViewModel.kt

```diff
@@ -248,11 +251,11 @@ public abstract class RainbowCakeViewModel<VS : Any>(initialState: VS) : ViewMod
         try {
             task()
         } catch (e: CancellationException) {
-            log(logTag, "Job cancelled exception:")
-            log(logTag, e)
+            log(logTag, "Job cancelled exception:", e, LogLevel.ERROR)
+            logError(logTag, e)
         } catch (e: Exception) {
-            log(logTag, "Unhandled exception in ViewModel:")
-            log(logTag, e)
+            log(logTag, "Unhandled exception in ViewModel:", e, LogLevel.ERROR)
```

Comment:
FYI this is my main purpose : activate `consumeExceptions` in production (to not have crashes) without catching Exceptions in all my ViewModels, but be able to monitor when we have errors.